plugins/README.md

```diff
@@ -0,0 +1,21 @@
+<!---
+Licensed under Creative Commons Attribution 4.0 International License
+https://creativecommons.org/licenses/by/4.0/
+--->
+
+# Fabric Plugins
+
+The current code base uses custom validation plugins for ecc (`ecc-vscc/`) and ercc (`ercc-vscc/`) to validate transactions. Additionally, ercc uses a decorator plugin (`ercc-decorator/`) to read attestation related data from the peer local filesystem during registration invocation.
+
+## Build plugins
+
+Go plugins are nasty little beasts and often cause pain when building. That is, a go plugin must be build with the same dependencies as used by the application that loads the plugin at runtime. Currently, Fabric still vendors dependencies which make building plugins even harder. So building plugins from the fabric project folder is the easiest way to overcome this at the moment.
+
+```
+$ ln -s $FPC_PATH/plugins $FABRIC_PATH/fpc_plugins
+$ cd $FABRIC_PATH/fpc_plugins; make
```

Comment:
i wonder whether that actually works with all shells and/or shell configurations (e.g., tcsh and cwdcmd command ..)

Reply:
seems also it is important that you do above _after_ you run `make provider` in ercc?

Reply:
it works at least in the fpc docker dev env. Did someone test with some other shell? if someone experiences a problem here we can create an issue and fix later;

Reply:
Oh, as mentioned in btw i'm fine leaving it, just thought i mention that it could lead to some potential problems (which we can fix later ...).
i guess the issue with make provider ordering you are solving now by trying to automate the whole build and symlink process in our makefiles, hence making manual instructions obsolete?